packages/core/.eslintrc.json

```diff
@@ -1,7 +1,8 @@
 {
     "env": {
         "node": true,
-        "mocha": true
+        "mocha": true,
+        "es6": true
```

Comment:
Just wondering what is the benefit about enable es6?

Reply:
Setting `es6` to `true` enables all of the ES6 globals such as Promise, Map, Set, etc. If not enabled, the eslint linter will raise the 'Promise not defined' error.